src/edu/ucsb/cs56/projects/games/dealer/BlackJackGui.java

```diff
@@ -398,7 +434,7 @@ public void display(Hand h){
                 gamePanel.repaint();
                 i+=200;
             }
-            sleep(500);
+            //sleep(500);
```

Comment:
Would you please add comment why a delay was used earlier and what was the problem for the delay? Add a comment so that anyone can understand in future.